### ISSUE TITLE
 Feature for "Detect/Split Scanned photos"

### ISSUE THREAD

**Issue by friishon** [Stirling-Tools/Stirling-PDF#1301]
Hi, 
I wanted to cut an image in half vertically, but I couldn't
Is it possible to add that function? 
I want to split scanned mangas which are in double page to have in single page, but i couldn't find a way to do.
And can be good if during the process modify the image to improve the quality of something

Thanks.

**Comment by Frooodle** [comment]
Sorry for late response would
https://stirlingpdf.io/split-pdf-by-sections
not do this?
![image](https://github.com/Stirling-Tools/Stirling-PDF/assets/77850077/58c07f2c-fae6-4cc1-96ef-9d7909ad4e25)

**Comment by friishon** [comment]
yes, but instead of pdf, doing for images, that section is only pdf
I would like to do in some mangas that has double page, and i want to split and then convert into a pdf or epub if is possible

**Comment by Frooodle** [comment]
Right now you'd have to do pipeline 

Image to pdf 
Split by sections 
And then can merge everything at end and do pdf to epub

**Comment by friishon** [comment]
i don't know how pipeline works

**Comment by Frooodle** [comment]
CLosing this as request is for images, but we are a PDF app
